pom.xml

```diff
@@ -27,23 +37,48 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-devtools</artifactId>
-            <scope>runtime</scope>
-            <optional>true</optional>
+            <artifactId>spring-boot-starter-validation</artifactId>
+            <version>3.1.5</version>
```

Comment:
you don't need to specify the version - it is taken from `spring-boot-starter-parent`